openseeds/ols-1/index.md

```diff
@@ -11,6 +11,21 @@ redirect_from:
   - /ols-1
 ---
 
+<div class="columns columns-toc">
+<div class="column is-one-fifth" id="cohort-toc" markdown="1">
+
+### Table of content
+{:.no_toc}
+
+<aside class="menu" markdown="1">
+
+- TOC
+{:toc .menu-list}
+
+</aside>
+</div>
```

Comment:
To limit redundancy, you could put this part in an file in `_include` folder

Reply:
I did that. It didn't work either. The output was the same as when I used a _cohort_ layout...